synthpop.proto

```diff
@@ -164,7 +164,13 @@ enum PwkStat {
 }
 
 message Health {
+    // Sourced from the Health Survey for England 2019
     BMI bmi = 1;
+    // See
```

Comment:
Using -1 as the sentinel value is awkward.

I may vote for switching from proto3 to proto2, which properly has optional fields, or checking if flatbuffers / something else has better support here.

Reply:
Agree